src/EditorFeatures/CSharpTest/Diagnostics/AddExplicitCast/AddExplicitCastTests_FixAllTests.cs

```diff
@@ -44,7 +44,7 @@ Base ReturnBase()
         return b;
     }
 
-    Base ReturnBase(Derived)
+    Base ReturnBase(Derived d)
```

Comment:
nit: Why add an unused named parameter?

Reply:
Oh I think the syntax is not right, I invoked it in another function:)